src/api/api-client/isi-geodata-eai/docs/ControllerZurExtraktionVonFeatureAufBasisVonMultiPolygoneImStandardEPSG4326WGS84Api.md

```
@@ -148,6 +149,72 @@ No authorization required
+### Authorization
+
+No authorization required
+
+### HTTP request headers
+
+- **Content-Type**: `application/json`
+- **Accept**: `application/hal+json`
+
+
+### HTTP response details
+| Status code | Description | Response headers |
```

markdownlint-cli2 (0.23.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)